exercises/binary-search/binary_search_test.rb

```diff
@@ -1,49 +1,81 @@
 require 'minitest/autorun'
 require_relative 'binary_search'
 
+# Common test data version: 1.3.0 bfb218f
 class BinarySearchTest < Minitest::Test
-  def test_it_has_list_data
+  def test_finds_a_value_in_an_array_with_one_element
+    # skip
+    binary = BinarySearch.new([6])
+    assert_equal 0, binary.search_for(6)
+  end
+
+  def test_finds_a_value_in_the_middle_of_an_array
+    skip
     binary = BinarySearch.new([1, 3, 4, 6, 8, 9, 11])
-    assert_equal [1, 3, 4, 6, 8, 9, 11], binary.list
+    assert_equal 3, binary.search_for(6)
   end
 
-  def test_it_raises_error_for_unsorted_list
```

Comment:
This test should still exist somewhere.

Reply:
I take that back.  This test should not exist... 

>    "Following https://github.com/exercism/problem-specifications/issues/234 the exercise",
    "should NOT include checking whether the array is sorted as it defeats",
    "the point of the binary search.",